nixos/lib/testing-python.nix

```diff
@@ -161,7 +161,12 @@ rec {
 
             echo -n "$testScript" > $out/test-script
             ${lib.optionalString (!skipLint) ''
-              ${python3Packages.pyflakes}/bin/pyflakes $out/test-script
+              ( # Define the variables
+                echo "def testScript(subtest, start_all, ${lib.concatStringsSep ", " nodeNames}):"
```

Comment:
Hm, I wonder whether there is a more elegant way to define this, since this could case all kinds of trouble we probably aren't aware of right now. One (pretty bad) example would be if you were doing something like this:

```
{
  testScript = ''
    machine.succeed('something
      where spaces are important')
  '';
}

Reply:
We need to do something. Defining them as variables might also work, but a linter may think too much of those variables.

We'll see. It doesn't alter the final script, so it can only cause a false error in the linter.

Reply:
One issue with this is that line numbers are no longer accurate.

Reply:
Okay, if we'd use `pyflakes.checker.Checker` directly, we should be able to directly provide a scope. Trying...

Reply:
@roberth: After digging through the code, I've found `PYFLAKES_BUILTINS`, which IMO is a better solution. I've added this as a squash-commit.